src/backend/src/controllers/reimbursement-requests.controllers.ts

```diff
@@ -126,4 +126,16 @@ export default class ReimbursementRequestsController {
       next(error);
     }
   }
+
+  static async getSingleReimbursementRequest(req: Request, res: Response, next: NextFunction) {
+    try {
+      const { requestId } = req.params;
```

Comment:
shouldn't we want to get req.params.reimbursementRequestId and use that to find the single reimbursement request

Reply:
that's what he's doing here, they're named requestId in this file for the sake of brevity, and the call to actually find the reimbursement request is in the service method